dateparser/conf.py

```diff
@@ -193,6 +196,27 @@ def check_settings(settings):
         'PREFER_LOCALE_DATE_ORDER': {
             'type': bool
         },
+
+        'DEFAULT_LANGUAGE': {
+            'type': list
+        },
+
+        'LANGUAGE_DETECTION_ENABLED': {
+            'type': bool
+        },
+        'LANGUAGE_DETECTION_EXTERNAL': {
+            'type': bool
+        },
+        'LANGUAGE_DETECTION_METHOD': {
+            'type': str
+        },
+
```

Comment:
I think you can remove the empty line between `LANGUAGE_DETECTION_METHOD` and `LANGUAGE_DETECTION_STRICT_USE`.